apps/website/src/content/docs/progressindicator.mdx

```diff
@@ -1,29 +1,90 @@
 ---
 title: Progress indicator
-description: Progress indicators express an unspecified wait time or display the length of a process.
+description: Progress indicators express the length of in-progress operations.
 thumbnail: ProgressIndicator
 ---
 
 <p>{frontmatter.description}</p>
 
-## Progress linear
+## Usage
 
-<Placeholder componentPageName='progresslinear--determinate' />
+There are two types of progress indicators which are linear and radial. Linear indicators show progress on a linear bar, while radial indicators display progress in a circular form.
+
+The default progress indicator is indeterminate. If a [value](#value) is passed, the indicator becomes determinate.
 
 <LiveExample src='ProgressLinear.main.jsx' truncate={false}>
   <AllExamples.ProgressLinearMainExample client:load />
 </LiveExample>
 
-### Props
+<LiveExample src='ProgressRadial.main.jsx' truncate={false}>
+  <AllExamples.ProgressRadialMainExample client:load />
+</LiveExample>
 
-<PropsTable path='@itwin/itwinui-react/esm/core/ProgressIndicators/ProgressLinear.d.ts' />
+### Value
 
-## Progress radial
+To specify a value for indeterminate indicator, pass a number into the `value` prop. This prop is available for both linear and radial indicators. If the `indeterminate` prop is set to `true`, this value will be ignored. This value must be inclusively between 0 and 100.
 
-<Placeholder componentPageName='progressradial--determinate' />
+<LiveExample src='ProgressLinear.value.jsx' truncate={false}>
+  <AllExamples.ProgressLinearValueExample client:load />
+</LiveExample>
 
-<LiveExample src='ProgressRadial.main.jsx' truncate={false}>
-  <AllExamples.ProgressRadialMainExample client:load />
+<LiveExample src='ProgressRadial.value.jsx' truncate={false}>
+  <AllExamples.ProgressRadialValueExample client:load />
+</LiveExample>
+
+Exclusively for linear indicators, the `isAnimated` flag is helpful when applying animation to the indicator as the value changes.
+
+<LiveExample src='ProgressLinear.isAnimated.jsx'>
+  <AllExamples.ProgressLinearAnimatedExample client:load />
+</LiveExample>
+
+### Status
+
+Three available statuses for both linear and radial progress indicators are:
+
+- `"positive"`
+- `"negative"`
+- `"warning"`
+
+<LiveExample src='ProgressLinear.status.jsx'>
+  <AllExamples.ProgressLinearStatusExample client:load />
+</LiveExample>
+
+Each radial indicator with status is accompanied by an icon, providing a visual cue to inform the current progress state better. The icon can be replaced by a [content](#content).
```

Comment:
Is this paragraph redundant? Feels a bit out of place in the "Status" section.

Reply:
I have made the change.